src/test/java/reposense/template/GitTestTemplate.java

```diff
@@ -28,7 +28,7 @@
  * Contains templates for git testing.
  */
 public class GitTestTemplate {
-    protected static final String TEST_REPO_GIT_LOCATION = "https://github.com/reposense/testrepo-Alpha.git";
+    protected static final String TEST_REPO_GIT_LOCATION = "https://github.com/chan-j-d/testrepo-Alpha.git";
```

Comment:
Should be changed back?

Reply:
Yep! Thanks for the reminder. Was leaving it in to show the test cases passing.